### PR TITLE
Add pydoc for BaseSensor

### PR DESCRIPTION
**Pull request by armab**
Very tiny cosmetic change for better IDE support when writing [new Sensors](http://docs.stackstorm.com/sensors.html#authoring-a-sensor)

autocomplete-ide-support

### CODE REVIEW

**Comment by manasdk** [comment]
+1

File: st2reactor/st2reactor/sensor/base.py [hunk 1]
```diff
@@ -16,6 +17,13 @@ class BaseSensor(object):
     """
 
     def __init__(self, sensor_service, config=None):
+        """
+        :param sensor_service: Sensor Service instance.
+        :type sensor_service: ``SensorService``
```
Comment (Kami) [inline st2reactor/st2reactor/sensor/base.py]: Ideally it should say ":class:`SensorService`"

Comment (armab) [inline st2reactor/st2reactor/sensor/base.py]: Nice thing, added that as well: https://libcloud.readthedocs.org/en/latest/development.html#docstring-conventions

However curious, is there any difference?

Also is there any difference between single and double quotes: 
```
``dict``
```
and
```
`dict`
```
?

**Comment by armab** [comment]
Again cleaned my messy history of commits with force push

**Comment by Kami** [comment]
Merged, thanks.

**Pull request merged**